typescript/utils/src/types.ts

```diff
@@ -5,6 +5,7 @@ export enum ProtocolType {
   Ethereum = 'ethereum',
   Sealevel = 'sealevel',
   Cosmos = 'cosmos',
+  Starknet = 'starknet',
```

Comment:
imo it'd be good if you could also add a starknet explorer family type since it looks like most starknet-chains share a similar explorer type https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/2cdeff43df7aba45d014dfe8989b1ae1a92f7ecd/typescript/sdk/src/metadata/chainMetadataTypes.ts#L21